### DISCUSSION TITLE
GitHub Code tab

### DISCUSSION THREAD

**Discussion by amalbenny** [community/community#48377] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Why does GitHub public repositories no longer showing 'code' tab. I was trying to learn GitHub Codespace with your tutorial in docs.github.com page. But how can I continue if I can't even connect with Codespace

**Comment by Kaparrs** [comment]
It does?

![image](https://user-images.githubusercontent.com/71441748/221224175-d3623e96-8cbe-4fa3-8c66-73318603fb3b.png)

**Comment by amalbenny** [comment]
![Screenshot_2023-02-25-19-38-02-32_e4424258c8b8649f6e67d283a50a2cbc](https://user-images.githubusercontent.com/119550625/221361402-afae947a-b485-4e24-a5c4-1eb19daa6e9c.jpg)

**Reply by Kaparrs** [reply]
Its because you're on mobile
